Cargo.toml

```diff
@@ -66,6 +66,10 @@ path = "src/main.rs"
 [profile.dev.package."*"]
 opt-level = 3
 
+[profile.test]
+opt-level = 3
```

Comment:
isn't this obfuscate backtrace and debug information? 
how do you think about running tests with release build job if this is desired?

Reply:
It should still keep debug info I'm pretty sure - the default for test enables debuginfo and I'm pretty sure it's orthogonal to opt-level: https://doc.rust-lang.org/nightly/cargo/reference/profiles.html#test

Reply:
When opt-level is not 0, the inlined callstacks are omitted, the progress is not linear to the code and some breakpoint doesn't work.

Reply:
I think the default development environment is better to be friendly to programmers unless there are unavoidable reasons